apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1775,17 +1775,16 @@ block_txs_list_by_hash_invalid_range(_Config) ->
     ok.
 
 naming_system_manage_name(_Config) ->
```

Comment:
* Would it be possible to add equivalent test with client specifying name as punycode rather than Unicode?

* Would it be possible to add test with two concurrent name pre-claim / claim going on, with one client specifying name as punycode and other as Unicode?

(In separate subsequent PR I mean.)

Reply:
Yes, this is feasible, but IMO such this is not a good idea, as such test will become stale.

`xn--` is a prefix of all punycode encoded domains. Based on our spec https://github.com/aeternity/protocol/blob/master/AENS.md#name, we should be using `CheckHyphens=true` config, which must satisfy the below:
```
If CheckHyphens, the label must not contain a U+002D HYPHEN-MINUS character in both the third and fourth positions.
```
See http://unicode.org/reports/tr46/#Validity_Criteria.

So my understanding is that our IDNA lib will not accept punycode domains (names starting with `xn--`) once it has proper validation implemented.

Reply:
> So my understanding is that our IDNA lib will not accept punycode domains (names starting with xn--) once it has proper validation implemented.

I suggest adding this as a test eventually (not this PR).